apps/mark-scan/backend/src/app.test.ts

```diff
@@ -362,20 +331,116 @@ test('printing a ballot increments the printed ballot count', async () => {
 });
 
 test('empty ballot box requires poll worker auth', async () => {
-  await configureMachine(mockUsbDrive, electionGeneralDefinition);
+  await configureForTestElection();
 
-  await expect(apiClient.confirmBallotBoxEmptied()).rejects.toThrow(
-    'Expected pollworker auth'
-  );
+  await suppressingConsoleOutput(async () => {
+    await expect(apiClient.confirmBallotBoxEmptied()).rejects.toThrow(
+      'Expected pollworker auth'
+    );
+  });
 });
 
 test('empty ballot box', async () => {
-  await configureMachine(mockUsbDrive, electionGeneralDefinition);
-  mockPollWorkerAuth(electionGeneralDefinition);
+  await configureForTestElection();
+  mockPollWorkerAuth(mockAuth, electionGeneralDefinition);
 
   await apiClient.confirmBallotBoxEmptied();
   expect(logger.log).toHaveBeenLastCalledWith(
     LogEventId.BallotBoxEmptied,
     'poll_worker'
   );
 });
+
+test('getPaperHandlerState returns state machine state', async () => {
+  await configureForTestElection();
+  await apiClient.setAcceptingPaperState();
+  expect(await apiClient.getPaperHandlerState()).toEqual('accepting_paper');
+});
+
+test('setAcceptingPaperState is a no-op when SKIP_PAPER_HANDLER_HARDWARE_CHECK flag is on', async () => {
+  expect(await apiClient.getPaperHandlerState()).toEqual('not_accepting_paper');
+  featureFlagMock.enableFeatureFlag(
+    BooleanEnvironmentVariableName.SKIP_PAPER_HANDLER_HARDWARE_CHECK
+  );
+  await apiClient.setAcceptingPaperState();
+  expect(await apiClient.getPaperHandlerState()).toEqual('not_accepting_paper');
+});
+
+test('printBallot sets the interpretation fixture when SKIP_PAPER_HANDLER_HARDWARE_CHECK is on', async () => {
+  await configureForTestElection();
+
+  featureFlagMock.enableFeatureFlag(
+    BooleanEnvironmentVariableName.SKIP_PAPER_HANDLER_HARDWARE_CHECK
+  );
+  await apiClient.printBallot({ pdfData: Buffer.of() });
+  await waitForExpect(async () => {
+    expect(await apiClient.getPaperHandlerState()).toEqual('presenting_ballot');
+  });
+  const interpretation = await apiClient.getInterpretation();
+  assert(interpretation);
+  expect(interpretation.type).toEqual('InterpretedBmdPage');
+  expect(interpretation.votes['102']).toEqual(['measure-102-option-yes']);
+});
+
+test('getInterpretation returns null if no interpretation is stored on the state machine', async () => {
+  expect(await apiClient.getInterpretation()).toEqual(null);
+});
+
+async function waitForExpectStatus(status: string): Promise<void> {
+  await waitForExpect(async () => {
+    expect(await apiClient.getPaperHandlerState()).toEqual(status);
+  });
+}
+
+function setPaperPresenting() {
+  driver.setPaperHandlerStatus({
+    paperInputLeftInnerSensor: true,
+    paperInputRightInnerSensor: true,
+    paperInputLeftOuterSensor: true,
+    paperInputRightOuterSensor: true,
+    paperNotPresent: false,
+    paperOutSensor: false,
+    paperJam: false,
+  });
+}
```

Comment:
Going to take a second look at this to see if `jest` can be used instead of the mock driver